### PR TITLE
Create "Clamp" Node

### PR DESCRIPTION
**Pull request by mrjschulte**
Add a new node to the chaiNNer toolset that will allow the user to Clamp the pixel ranges in an image to a user defined Minimum and Maximum value.

### CODE REVIEW

File: backend/src/packages/chaiNNer_standard/image_adjustment/adjustments/clamp.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    inputs=[
+        ImageInput(),
+        SliderInput(
+            "Minimum",
+            minimum=0.0,
+            maximum=1.0,
+            default=0.0,
+            precision=4,
+            controls_step=0.001,
+            scale="log",
```
Comment (RunDevelopment) [inline backend/src/packages/chaiNNer_standard/image_adjustment/adjustments/clamp.py:25]: Does `log` make sense here? Wouldn't a linear scale better?

File: backend/src/packages/chaiNNer_standard/image_adjustment/adjustments/clamp.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    schema_id="chainner:image:clamp",
+    name="Clamp",
+    description="Clamps the values of an image.",
+    icon="ImContrast",
+    inputs=[
+        ImageInput(),
+        SliderInput(
+            "Minimum",
+            minimum=0.0,
+            maximum=1.0,
```
Comment (RunDevelopment) [inline backend/src/packages/chaiNNer_standard/image_adjustment/adjustments/clamp.py:21]: Same old: 0-1, 0-100, or 0-255? What do you think would be best here?

File: backend/src/packages/chaiNNer_standard/image_adjustment/adjustments/clamp.py (hunk 1 above)
Comment (mrjschulte) [reply backend/src/packages/chaiNNer_standard/image_adjustment/adjustments/clamp.py:25]: I generally like the log scale for my sliders as it feels more exact at the lower end.

File: backend/src/packages/chaiNNer_standard/image_adjustment/adjustments/clamp.py (hunk 2 above)
Comment (mrjschulte) [reply backend/src/packages/chaiNNer_standard/image_adjustment/adjustments/clamp.py:21]: I think we should slowly consider a move to a normalized / float (0.0 - 1.0 in this case) Ui paradigm for denoting image bit-depth and applicable and relevant operational value ranges.

Comment (RunDevelopment) [reply backend/src/packages/chaiNNer_standard/image_adjustment/adjustments/clamp.py:21]: Man, we really need a way for users to select the unit they want, and we always get 0-1. 

Let's keep it as is for now.

**Review by RunDevelopment — APPROVED** [review]

**Pull request merged**